### PR TITLE
Intake form controller

### PR DESCRIPTION
**Pull request by kwestein**
Intake form - no validations and still ugly.

@OttawaU-RSSP/team

### CODE REVIEW

File: app/controllers/intake_forms_controller.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+class IntakeFormsController < ApplicationController
+  def new
+  end
+
+  def create
+    sponsor_group = SponsorGroup.create(sponsor_group_params)
```
Comment (garymardell) [inline app/controllers/intake_forms_controller.rb]: `.new` and then `.save` on the if.

File: app/controllers/intake_forms_controller.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      redirect_to root_path
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def sponsor_group_params
+    @sponsor_group_params ||= params.require(:intake_form).permit(
```
Comment (garymardell) [inline app/controllers/intake_forms_controller.rb]: No need to memoize the params as they are only used once, only memoize when required.

File: app/models/sponsor_group.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class SponsorGroup < ActiveRecord::Base
+  has_one :application
+
+  validates :citizenship_status, inclusion: { in: ["Canadian Citizen", "Permanent Resident", "Other"] }
```
Comment (garymardell) [inline app/models/sponsor_group.rb]: Bike shed but could move these into a constant.

File: app/controllers/intake_forms_controller.rb (hunk 1 above)
Comment (garymardell) [inline app/controllers/intake_forms_controller.rb]:
``` ruby
class IntakeForm
  include ActiveModel::Model

  validates :email, presence: true

  def save
    SponsorGroup.create(attribute)
  end
end
```

**Comment by kwestein** [comment]
@garymardell wanna take a  look at the last commit?

**Comment by garymardell** [comment]
:+1:

**Pull request merged**